examples/sandbox/index.ts

```diff
@@ -48,17 +48,18 @@ const keepkeyAdapter = WebUSBKeepKeyAdapter.useKeyring(keyring);
 const kkemuAdapter = TCPKeepKeyAdapter.useKeyring(keyring);
 const portisAdapter = PortisAdapter.useKeyring(keyring, { portisAppId });
 const nativeAdapter = NativeAdapter.useKeyring(keyring, {
-  mnemonic,
   deviceId: "native-wallet-test",
 });
 
 const log = debug.default("hdwallet");
 
 keyring.onAny((name: string[], ...values: any[]) => {
+  console.log("Event:", name, values);
   const [[deviceId, event]] = values;
   const { from_wallet = false, message_type } = event;
   let direction = from_wallet ? "🔑" : "💻";
   debug.default(deviceId)(`${direction} ${message_type}`, event);
+  alert(`Event: ${name[2]}`);
```

Comment:
Probably want to remove this debugging?

Reply:
Well, it's for the sandbox and I found it really helpful to show when an event was happening. I suppose I could have it just update some text on the page somewhere.

Reply:
Yeah I would rather it not create an alert box on any event personally. Do you know if the `debug.default` logs out the events to console as is already?

Reply:
I went ahead and put some time into implementing a mnemonic_required modal in sandbox and added an "event log" section rather than relying on "alert".